kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/coordinatedcommits/CommitCoordinatorProviderSuite.scala

```diff
@@ -200,7 +200,7 @@ class Builder2(hadoopConf: Configuration) extends CommitCoordinatorBuilder(hadoo
 // Test 3
 // Commit Coordinator Client with semanticEquals implemented for testing
 class TestCommitCoordinatorClient3(val key: String) extends TestCommitCoordinatorClientBase {
-  override def semanticEquals(other: CommitCoordinatorClient): lang.Boolean =
+  override def semanticEquals(other: CommitCoordinatorClient): Boolean =
```

Comment:
[nit] JBoolean

Reply:
I will fix all the imports in a follow up.